### DISCUSSION TITLE
Solve JSX & Async Server Components

### DISCUSSION THREAD

**Discussion by Anerco** [mattpocock/ts-reset#141] [Ideas]
in next js, rendering an async server component results in the type error:
SomeComponent cannot be used as a JSX component.
  Its return type 'Promise<Element>' is not a valid JSX element.

is there any way we can solve it using ts-reset?

**Comment by mattpocock** [comment]
Unfortunately not! But this is being resolved in TS 5.1 I think.

**Reply by Anerco** [reply]
Thanks Matt!
